tests/longevity/results/1.1.0/1.1.0.md

```diff
@@ -93,21 +93,23 @@ severity=ERROR
 SEARCH("error")
 ```
 
-There were 36 error logs across 2 pod instances. They came in 2 almost identical batches, one on the first day of
-running the test, after approximately 6 hours, and the second 2.5 days later. They were both relating to leader election
-loss, and subsequent restart (see https://github.com/nginxinc/nginx-gateway-fabric/issues/1100).
-
-Both error batches caused the pod to restart, but not terminate. However, the first pod was terminated about 10 minutes
-after the first error batch and subsequent restart occurred. Exactly why this pod was terminated is not currently clear,
-but it looks to be a cluster event (perhaps an upgrade) as the coffee and tea pods were terminated at that time also.
-Strangely, there were 6 pod restarts in total of the second pod, but no other errors were observed over the test period
-other than what was seen above, and grepping the logs for start-up logs only produced the 2 known restarts relating to
-the leader election loss, plus initial start-up of both pods (4 in total).
-
-```console
-kubectl get pods -n nginx-gateway
-NAME                                               READY   STATUS    RESTARTS      AGE
-my-release-nginx-gateway-fabric-78d4b84447-4hss5   2/2     Running   6 (31h ago)   3d22h
+There were 53 error logs across 3 pod instances. They came in 3 almost identical batches, starting just over 24 hours
```

Comment:
I'm slightly confused why we have 3 pod instances, not one 🤔 
Is that because they were terminated? But in this case, you also write "Each error batches caused the pod to restart, but not terminate."

Reply:
I've reworded the section a bit which hopefully makes it a little more clear.